scripts/test.sh

```diff
@@ -2,7 +2,5 @@
 
 set -e
 
-rm -rf "node_modules/"
```

Comment:
What did it solve?

Reply:
Before this change i was unable to run test scripts - `npm run test:webpack1`, etc. because the script installed only `webpack` and all dev dependencies such as `multi-loader` were missing. Additionally, running subsequent tests is now quicker because there is no need to reinstall common dependencies. Please correct me if i am missing something.